$NetBSD: patch-ad,v 1.3 2013/01/20 06:53:53 mef Exp $

Avoid following problem:
 ----
In file included from gravity.c:19:0:
../config.h:562:9: warning: conflicting types for built-in function 'alloca'
In file included from ../config.h:577:0,
                 from gravity.c:19:
/usr/include/stdlib.h:250:7: error: conflicting types for 'alloca'
../config.h:562:9: note: previous declaration of 'alloca' was here
 ----

--- config.h.in.orig	2006-12-09 11:54:06.000000000 +0000
+++ config.h.in	2006-12-12 08:52:22.000000000 +0000
@@ -432,6 +432,8 @@
 #else
 #  ifdef _AIX
        #pragma alloca
+#  elif defined(__DragonFly__) || defined(__FreeBSD__) || defined(__NetBSD__)
+#	include <stdlib.h>
 #  else
 #    ifndef alloca /* predefined by HP cc +Olibcalls */
	 char *alloca ();
